js/custom.js

```diff
@@ -2,7 +2,7 @@
 
   $(document).ready(function(){
 
-    $('.sky-part').css({minHeight: $(window).height()});
+    $('.sky-part:not(.sky-part-v2)').css({minHeight: $(window).height()});

```

Comment:
Can be done with CSS3 '100vh', but anyway can't find selector for element.
Should't this be modified for resize event below as well?